dji_sdk/gimbal gimbal_sp
---
bool success